src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cleared = sourceString.split(';');
+
+  return cleared.reduce((accum, style) => {
+    const n = style
+      .split(':')
+
+      .map((element) => element.trim());
```

Comment:
i think we can do it without map
At each iteration u have a `key` and a `value`.
u can work with them without additional methods.
`I'll leave a comment below for better understanding`